config/src/main/java/org/springframework/security/config/annotation/web/configurers/ExpressionUrlAuthorizationConfigurer.java

```diff
@@ -108,6 +109,57 @@ public ExpressionInterceptUrlRegistry getRegistry() {
 			extends
 			ExpressionUrlAuthorizationConfigurer<H>.AbstractInterceptUrlRegistry<ExpressionInterceptUrlRegistry, AuthorizedUrl> {
 
+		private boolean isAnyRequestConfigured = false;
```

Comment:
Is there a reason to add this to `ExpressionUrlAUthorizationConfigurer` instead of `AbstractRequestMatcherRegistry`? I think it would be nice if all subclasses of `AbstractRequestMatcherRegistry` were able to inherit this behavior. What did I miss?

Reply:
If it's due to `mvcMatchers`, I think that you could add the check to `AbstractRequestMatcherRegistry#createMvcMatchers`.

Reply:
Also, let's leave `is` off the front and call it `anyRequestConfigured` instead.

Reply:
> Is there a reason to add this to `ExpressionUrlAUthorizationConfigurer` instead of `AbstractRequestMatcherRegistry`? I think it would be nice if all subclasses of `AbstractRequestMatcherRegistry` were able to inherit this behavior. What did I miss?

Fixed.

Reply:
> If it's due to `mvcMatchers`, I think that you could add the check to `AbstractRequestMatcherRegistry#createMvcMatchers`.

Fixed.

Reply:
> Also, let's leave `is` off the front and call it `anyRequestConfigured` instead.

Fixed